backend/src/main/java/com/redhat/cloud/notifications/routers/AdminService.java

```diff
@@ -31,15 +31,15 @@
 
     @GET
     @Produces(APPLICATION_JSON)
-    public Uni<Response> debugRbac(@QueryParam("rhid") String rhid) {
+    public Response debugRbac(@QueryParam("rhid") String rhid) {
 
-        Uni<RbacRaw> rbacRawUni = rbacServer.getRbacInfo("notifications,integrations", rhid);
-
-        return rbacRawUni
-                .onItem()
-                .transform(r -> Response.ok(r.data).build())
-                .onFailure()
-                .call(t -> Uni.createFrom().item(Response.serverError().entity("Rbac call failed -- see logs").build()));
+        try {
+            RbacRaw rbacRaw = rbacServer.getRbacInfo("notifications,integrations", rhid)
+                    .await().atMost(Duration.ofSeconds(2L));
```

Comment:
I think we have seen requests taking more than 2s? or 2s was the max allowed?

Reply:
We currently time out after 2 seconds and the request is canceled. Also, this is only the `AdminService` which is rarely used.